### ISSUE TITLE
Todo example fails on fresh install

### ISSUE THREAD

**Issue by OverclockedTim** [facebook/relay#289]
npm install for the TODO example currently fails from a fresh git pull of master. The build script fails when attempting to preinstall react-relay since it cannot find babel-relay-plugin.  The babel-relay-plugin is in the root scripts directory and npm build of the root directory works fine. It is the build of the TODO example only that fails. NPM output is below:

> react-relay@0.2.1 preinstall /Users/bluefire/Documents/workspaces/web/relay/examples/todo/node_modules/react-relay
> cd scripts/babel-relay-plugin && npm install

sh: line 0: cd: scripts/babel-relay-plugin: No such file or directory
npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3
npm ERR! code ELIFECYCLE

npm ERR! react-relay@0.2.1 preinstall: `cd scripts/babel-relay-plugin && npm install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-relay@0.2.1 preinstall script 'cd scripts/babel-relay-plugin && npm install'.
npm ERR! This is most likely a problem with the react-relay package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd scripts/babel-relay-plugin && npm install
npm ERR! You can get their info via:
npm ERR!     npm owner ls react-relay
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/bluefire/Documents/workspaces/web/relay/examples/todo/npm-debug.log

**Comment by zpao** [comment]
This looks like it should get fixed by #287. cc @josephsavona

**Comment by OverclockedTim** [comment]
Yes, when I applied the same revert as #287 locally, the example does indeed build, thanks!

**Comment by OverclockedTim** [comment]
Once #287 is applied, the todo application fails with the following in the console: 

Uncaught TypeError: Cannot read property 'unstable_batchedUpdates' of undefined
app.js:5728 Uncaught ReferenceError: Relay is not defined

Since it looks as though there are several issues with the examples, is there a better place to look at working code? A different branch perhaps?

**Comment by OverclockedTim** [comment]
For what it is worth, react-14-rc seems to be working...

**Comment by wincent** [comment]
```
Uncaught TypeError: Cannot read property 'unstable_batchedUpdates' of undefined
app.js:5728 Uncaught ReferenceError: Relay is not defined
```

Possibly related to https://github.com/facebook/relay/commit/2d822400dbcf14f9636ab3149477f4ff0a0ee466

Perhaps we need to update the React deps in the examples' `package.json` files?

**Comment by wincent** [comment]
> Perhaps we need to update the React deps in the examples' `package.json` files?

Which is already happening in https://github.com/facebook/relay/pull/290 I see, so I think we might be good here.

**Comment by steveluscher** [comment]
Sorry, @OverclockedTim. I went through all of the same step that you went through today, called for the revert in #287, and fixed the rest of the issues in #290. Should land soon! Thanks for your patience.

**Comment by OverclockedTim** [comment]
Thanks Steve!
